internal/security/setup/option/generate_test.go

```diff
@@ -170,47 +163,43 @@ func setupGenerateTest(t *testing.T) func(t *testing.T) {
 		t.Fatalf("cannot copy %s for the test: %v", configTomlFile, err)
 	}
 
-	setup.Init()
+	_ = setup.Init()
```

Comment:
is there a reason not to check err here too?

Reply:
I don't see this error being checked, when in the other file we do check the error from `Init`. Please check this error as well

Reply:
oh, thought i got it